warp/static/js/create/index.js

```diff
@@ -65,36 +65,9 @@ $(() => {
       appendSlide(v, i);
     });
     resizeSlides();
-  };
-
-  $('#create_html').submit((e) => {
-    const subject = $('#id_subject').val();
-    const isPublic = document.querySelector('#id_is_public').checked;
-    const slideList = editor.getValue().split(/={5,}/g).map((v, i) => ({
-      slide_order: i,
-      markdown: v.trim(),
-      html: marked(v.trim()),
-    }));
 
-    const presentation = {
-      subject,
-      is_public: isPublic,
-      slide_list: slideList,
-    };
-
-    $.ajax({
-      url: 'create',
-      method: 'POST',
-      data: presentation,
-    }).done(() => {
-      location.href = '/';
-    }).error((xhr, st, err) => {
-      console.log(xhr);
-      console.error(err);
-    });
-
-    e.preventDefault();
-  });
+    document.getElementById('id_markdown').value = editor.getValue();
```

Comment:
I think this line is error

I don't know why

editor id is `markdown_editor`, not `id_markdown`